submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+      <ol>
+         <li><a href="#">New Releases</a></li>
+	 <li><a href="#">General Book Shelf</a></li>
+	 <li><a href="#">Today's Deals</a></li>
+         <li><a href="#">Registration</a></li> 
+	 <li><a href="#">Teachers Corner</a></li>
+	 <li><a href="#">Students Corner</a></li>
+         <li><a href="#">Contact</a></li>
+	 <li><a href="#">Order Offline</a></li>
+      </ol>
+  </header>
+   <main>
+    <div>
```

Comment:
@Oluwabusiolami You did not close this `div`

Reply:
It is well. I have removed it now. I hope it will be approved now @adaugochi @ToluAkin